docs/3.1. Interoperability - NMOS IS-04.md

```diff
@@ -39,4 +39,4 @@ Where this Device creates a new Source at an Output (see [4.0 Behaviour](4.0.%20
 
 ## Version Increments
 
-In order to prevent unnecessary polling of the Audio Channel Mapping API, any changes to active map parameters belonging to Source connected Outputs MUST be signalled via the IS-04 versioning mechanism by means of a version timestamp increment.
+In order to prevent unnecessary polling of the Audio Channel Mapping API active map resource, any changes to active map parameters belonging to Source connected Outputs MUST be signaled via the IS-04 versioning mechanism by means of a version timestamp increment on the related "source" object.  For active map parameters belonging to outputs which are not backed by an IS-04 registered source, the version timestamp of the related IS-04 device object should be updated.
```

Comment:
```suggestion
In order to prevent unnecessary polling of the Audio Channel Mapping API active map resource, any changes to active map parameters belonging to Source connected Outputs MUST be signaled via the IS-04 versioning mechanism by means of a version timestamp increment on the related "Source" object.  For active map parameters belonging to outputs which are not backed by an IS-04 registered Source, the version timestamp of the related IS-04 Device object should be updated.
```

Reply:
I thought the agreement was to update the Device version timestamp regardless of the output landing as an NMOS source or not.

Reply:
In email conversation with John on 28 Feb we agreed that the Device version number must **always** be updated and the related Source (if one has been registered) is a new Source and the version number must also be updated.